filament/backend/include/backend/DriverEnums.h

```diff
@@ -1002,29 +979,37 @@ struct StencilState {
             uint8_t padding1                                : 2;                    // 16
 
             //! Masks the bits of the stencil values participating in the stencil comparison test.
-            uint8_t readMask                                : 8;                    // 24
+            uint8_t readMask;                                                       // 24
 
             //! Masks the bits of the stencil values updated by the stencil test.
-            uint8_t writeMask                               : 8;                    // 32
+            uint8_t writeMask;                                                      // 32
         };
         uint32_t u = 0;
     };
```

Comment:
Do we even need this `union` anymore?

Reply:
that's a good point, let me play with it. probably not.

Reply:
Note that this struct may grow—we can actually move the stencil reference value here, as it's also set separately per front/back facing polygons.